students/N0vA/lesson_03/mailroom.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+import sys
+#### Mailroom Project Part 1 ####
+
+# Create list of donors and their donation history
+database = [('Bill Gates', [2000000, 250000000]), ('Jeff Bezos', [2000000]), ('Elon Musk', [50000000, 10000000]), ('Howard Schultz', [1000000]), ('Paul Allen', [450000000])]
+
+# Send a thank you
+def thank_you():
+    
+    donor_list = database
+    print('Alright.  Which donor would you like to send a thank you card?')
+    
+    while True:
+        person = input('Enter their name here or type "list" to see a list of donors: ')
+        past_donor = False
+
+        # Print list option
+        if person.lower() == 'list':
+            past_donor = True
+            for per in donor_list:
+                print(per[0])
+
+        else:
+            for name in donor_list:
+                if person.lower() == name[0].lower():
+                    past_donor = True
+
+            # Adding a new donor
+            if past_donor == False:
+                amount_donated = input('How much was their donation? ')
+                amount_donated = int(amount_donated)
+                new_record = (person, [amount_donated])
+                donor_list = donor_list.append(new_record)
```

Comment:
the assignment asks to add or update existing donations. Your code is written to only add donation on new donation.